### PR TITLE
Update control-usb-devices-using-intune.md

### PR DESCRIPTION
**Pull request by msbemba**
Removed the Endpoint ATP baseline settings to control removable storage as Device control is no longer a part of baseline settings.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5458

**Review by JohanFreelancer9 — COMMENTED** [review]
Should "Use Microsoft Defender for Endpoint baseline settings" not be removed from the preceding table as well?

**Comment by msbemba** [comment]
@JohanFreelancer9  Thank you. Could you please copy review again?

File: microsoft-365/security/defender-endpoint/control-usb-devices-using-intune.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The following table describes the ways Microsoft Defender for Endpoint can allow
 | [Allow installation and usage of specifically approved peripherals with matching device instance IDs](#allow-installation-and-usage-of-specifically-approved-peripherals-with-matching-device-instance-ids) | You can only install and use approved peripherals that match any of these device instance IDs. |
 | [Prevent installation and usage of specifically prohibited peripherals with matching device instance IDs](#prevent-installation-and-usage-of-specifically-prohibited-peripherals-with-matching-device-instance-ids) | You can't install or use prohibited peripherals that match any of these device instance IDs. |
 | [Limit services that use Bluetooth](#limit-services-that-use-bluetooth) | You can limit the services that can use Bluetooth. |
-| [Use Microsoft Defender for Endpoint baseline settings](#use-microsoft-defender-for-endpoint-baseline-settings) | You can set the recommended configuration for ATP by using the Defender for Endpoint security baseline. |
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/control-usb-devices-using-intune.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder.

**Pull request merged**